code/game/turfs/turf.dm

```diff
@@ -420,9 +420,12 @@
 		if(istype(src, /turf/simulated))
 			var/turf/simulated/T = src
 			T.dirt = 0
+			T.color = null
 		for(var/obj/effect/O in src)
-			if(istype(O,/obj/effect/rune) || istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
+			if(istype(O,/obj/effect/decal/cleanable) || istype(O,/obj/effect/overlay))
```

Comment:
Removing runes from this will make runes drawn in crayon uncleanable, they use the same object. We don't want uncleanable graffiti

Introducing new bugs is not good, i'd advise seperating them out and making sure the crayon runes are cleanable before this is merged, maybe DNM til then, or maybe take out this change for now and do it in a seperate PR when you have time to fix it properly